core/controllers/domain_objects_validator.py

```diff
@@ -37,20 +38,27 @@ def validate_exploration_change(obj):
 
     Args:
         obj: dict. Data that needs to be validated.
+
+    Returns:
+        ExplorationChange. Returns ExplorationChange object after validation.
     """
     # No explicit call to validate_dict method is necessary, because
     # ExplorationChange calls validate method while initialization.
-    exp_domain.ExplorationChange(obj)
+    return exp_domain.ExplorationChange(obj)
 
 
-def validate_new_config_property_values(obj):
+def validate_new_config_property_values(new_config_property_value_dicts):
     # type: (Dict[String, Any]) -> None
     """Validates new config property values.
```

Comment:
The function name and docstring are wrong. They don't refer to dicts.

Reply:
Updated.